.changeset/pre.json

```diff
@@ -1,6 +1,6 @@
 {
   "mode": "pre",
-  "tag": "beta-v5",
+  "tag": "beta",
```

Comment:
🤔 Currently all the package.json files in this monorepo refer to `@beta-v5.{something}`. Should we add a patch version bump in this changeset to make them convert to `beta`?

Reply:
I think this is going to happen in the next Vesion Packages PR even if we don't do anything :)